scripts/zones/Misareaux_Coast/mobs/Gration.lua

```diff
@@ -31,7 +31,8 @@ entity.onMobFight = function(mob, target)
     for _, v in ipairs(enmityList) do
         -- repeatedly apply short duration intimidate (5 sec) with 50% power to all entities on enmity list
         if not v["entity"]:hasStatusEffect(xi.effect.INTIMIDATE) then
-            v["entity"]:addStatusEffectEx(xi.effect.INTIMIDATE, 0, 50, 3, 5)
+            -- Pass in the mobID as the status effect subID so can be checked
+            v["entity"]:addStatusEffectEx(xi.effect.INTIMIDATE, 0, 50, 3, 5, mob:getID())
```

Comment:
Why does gration not have a very high [Humanoid_Killer trait](https://www.bg-wiki.com/ffxi/Humanoid_Killer) 
This would cover everything but pets?  
And if avatars are also supposed to be intimidated - we could add an intimidate all mod

Reply:
> Why does gration not have a very high [Humanoid_Killer trait](https://www.bg-wiki.com/ffxi/Humanoid_Killer) This would cover everything but pets? And if avatars are also supposed to be intimidated - we could add an intimidate all mod

It looks like originally it was coded to intimidate within some area (20 yalms) of Gration (and thus needed the status effect) but was later changed to remove that part I think, a bit unclear.

Gration does not seem to intimidate jug pets at least (https://www.youtube.com/watch?v=6O30nGgmun0) or avatars (https://www.youtube.com/watch?v=w48VSY5g--o&t=212s), so maybe Humanoid Killer could work. Good suggestion as will simplify the code.

Reply:
Good chance whomever wrote it originally didnt know about humanoid_killer or humanoid_killer didnt exist yet or something else all together.

Reply:
> Good chance whomever wrote it originally didnt know about humanoid_killer or humanoid_killer didnt exist yet or something else all together.

ok, I updated the PR.